packages/integration-shims/src/launchDarkly.ts

```diff
@@ -0,0 +1,38 @@
+import { consoleSandbox, defineIntegration, isBrowser } from '@sentry/core';
+import { FAKE_FUNCTION } from './common';
+
+/**
+ * This is a shim for the LaunchDarkly integration.
+ * We need this in order to not throw runtime errors when accidentally importing this on the server through a meta framework like Next.js.
+ */
+export const launchDarklyIntegrationShim = defineIntegration((_options?: unknown) => {
+  if (!isBrowser()) {
+    consoleSandbox(() => {
+      // eslint-disable-next-line no-console
+      console.warn('The launchDarklyIntegration() can only be used in the browser.');
+    });
+  }
+
+  return {
+    name: 'LaunchDarkly',
+  };
+});
+
+/**
+ * This is a shim for the LaunchDarkly flag used handler.
+ */
+export function buildLaunchDarklyFlagUsedHandlerShim(): unknown {
+  if (!isBrowser()) {
+    consoleSandbox(() => {
+      // eslint-disable-next-line no-console
+      console.warn('The buildLaunchDarklyFlagUsedHandler() should only be used in the browser.');
```

Comment:
```suggestion
      console.warn('The buildLaunchDarklyFlagUsedHandler() can only be used in the browser.');
```
To align the messages with the previous one.